source/renderer/app/components/status/DaedalusDiagnostics.tsx

```diff
@@ -26,6 +26,8 @@ import type { CoreSystemInfo } from '../../types/coreSystemInfoTypes';
 import type { TipInfo } from '../../api/network/types';
 import { ErrorType } from '../../domains/ApiError';
 
+const popupZIndex = 10000;
```

Comment:
What if we move this to `TopLevelPopover` and map the value to a layer? E.g 
'foreground' -> 10000 | 'dialog' -> 9999

Reply:
Decided to scrap this because there's a simpler way (unless we go from polymorph).

Unfortunately, with TopLevelPopover, we still have the problem of Input fields (which are using tippy from polymorph) to have a 9999 z-index (and so . So the simpler way is to define a separate z-index for news feed (10000) and have the other dialogs at 9998 while keeping the default PopOvers (9999).

This makes the news feed over all popovers in case width of the screen is small enough for them to overlap.